app/models/passenger.rb

```diff
@@ -0,0 +1,19 @@
+class Passenger < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true
+  validates :phone_num, presence: true
+
+  def total_spent
+    total = 0
+    self.trips.each do |trip|
+      total += trip.cost.to_i
+    end
+    return total
+  end
```

Comment:
This can be simplified with a `sum`:

```suggestion
  def total_spent
    return self.trips.sum do |trip|
      trip.cost.to_i
    end
  end
```